docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,11 +6,14 @@
 
 ## Checklist:
 
-_these will eventually be enforced by CircleCI; we need to set that up at the USDOJ org level_
```

Comment:
Removing this line because these are now set up and enforced at the USDOJ org level!  🎉